demisto_sdk/commands/common/hook_validations/docker.py

```diff
@@ -42,36 +40,44 @@ def is_docker_image_valid(self):
         if not self.docker_image_latest_tag:
             self.is_valid = False
         elif not self.is_docker_image_latest_tag():
-            print_error(Errors.not_latest_docker(self.file_path, self.docker_image_tag, self.docker_image_latest_tag))
             self.is_valid = False
         return self.is_valid
 
     def is_docker_image_latest_tag(self):
+        if 'demisto/python:1.3-alpine' == f'{self.docker_image_name}:{self.docker_image_tag}':
+            # the docker image is the default one
+            self.is_latest_tag = False
+            print_error('The current docker image in the yml file is the default one: demisto/python:1.3-alpine,\n'
+                        'Please create or use another docker image\n')
+            return self.is_latest_tag
+
         if not self.docker_image_name or not self.docker_image_latest_tag:
             # If the docker image isn't in the format we expect it to be or we failed fetching the tag
             # We don't want to print any error msgs to user because they have already been printed
             self.is_latest_tag = False
             return self.is_latest_tag
 
-        server_version = LooseVersion(self.from_version)
-        # Case of a modified file with version >= 5.0.0
-        if self.is_modified_file and server_version >= '5.0.0':
-            if self.docker_image_latest_tag != self.docker_image_tag and not \
-                    'demisto/python:1.3-alpine' == '{}:{}'.format(self.docker_image_name, self.docker_image_tag):
-                # If docker image name are different and if the docker image isn't the default one
-                self.is_latest_tag = False
-        # Case of an added file
-        elif not self.is_modified_file:
-            if self.docker_image_latest_tag != self.docker_image_tag:
-                self.is_latest_tag = False
+        if self.docker_image_latest_tag != self.docker_image_tag:
+            # If docker image tag is the most updated one that exists in docker-hub
+            self.is_latest_tag = False
 
         if not self.is_latest_tag:
-            print_error('The docker image tag is not the latest, please update it.\n'
-                        'The docker image tag in the yml file is: {}\n'
-                        'The latest docker image tag in docker hub is: {}\n'
+            self.is_latest_tag = False
+            print_warning('The docker image tag is not the latest, please update it.\n'
```

Comment:
do we have a test that checks this warning is printed?

Reply:
yes it's in test_negative__non_latest_docker_image in validatie_integration_test.py